content/docs/concepts/security/index.md

```diff
@@ -771,6 +770,56 @@ spec:
     name: "products-viewer"
 {{< /text >}}
 
+### Using Istio authorization on plain TCP protocols
+
+The examples in [ServiceRole](#servicerole) and [ServiceRoleBinding](#servicerolebinding) show a
```

Comment:
Please change the headings of the sections to Service role and Service role binding respectively. Do not use the value of `kind` in the configuration files to refer to the files themselves or worse, the configuration objects. Change "a" to "the": ... show the